documentation/source/user_section/tutorials/shape-analysis/compute-csa-and-other-shape-metrics/other-shape-metrics.rst

```diff
@@ -3,11 +3,23 @@
 Other shape metrics
 ###################
 
-The csv files generated by ``sct_process_segmentation`` also include metrics to analyse the shape of the spinal cord in the axial plane, such as ellipticity, antero-posterior and right-left dimensions. These are of particular interest for studying cord compression. See `Martin et al. BMJ Open 2018 <https://bmjopen.bmj.com/content/8/4/e019809>`_ for an example application in degenerative cervical myelopathy.
+The csv files generated by ``sct_process_segmentation`` include besides the cross-sectional area (CSA) also the following metrics to analyse the shape of the spinal cord in the axial plane:
+
+- antero-posterior diameter (AP diameter), ``diameter_AP``
+- transverse diameter, ``diameter_RL``
+- eccentricity, ``eccentricity``
+- solidity, ``solidity``
+- orientation, ``orientation``
+
+These are of particular interest for studying cord compression. See `Martin et al. BMJ Open 2018 <https://bmjopen.bmj.com/content/8/4/e019809>`_ and `Horáková et al. Quant Imaging Med Surg 2022 <https://pubmed.ncbi.nlm.nih.gov/35371944/>`_ for an example applications.
+
+For an interactive database of normative values, see `Valošek, Bédard et al. NeuroLibre 2023 <https://neurolibre.org/papers/10.55458/neurolibre.00017>`_.
 
 .. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/shape-metric-computation/sct_process_segmentation-shape-metrics.png
    :align: center
 
+   Figure Caption: (A) Compression at the level of the intervertebral disc C5-C6 (red arrows) is manifested by (B) the decrease of AP Diameter, Compression Ratio, and Solidity and by the increase of Eccentricity (red arrows). The blue line represents normative values computed from 105 males from the spine-generic dataset. Adapted from `Valošek, Bédard et al. NeuroLibre 2023 <https://doi.org/10.55458/neurolibre.00017>`__.
+
```

Comment:
Because of the lengthy caption (and the lack of caption styling from our theme), I'm now finding the page a little hard to read:

> ![Screenshot from 2023-10-19 11-55-54](https://github.com/spinalcordtoolbox/spinalcordtoolbox/assets/16181459/e6684786-cb2a-4749-8e43-3d9cc6559d6f)

It is hard for me to tell at a glance which text is related to the figure, which text is related to the table, and which text is just normal "page text". As well, it sort of looks like a "wall of text" to me.

To improve this, we could add custom CSS styling for figure captions (italic, smaller text). See:

> ![image](https://github.com/spinalcordtoolbox/spinalcordtoolbox/assets/16181459/a85c2fce-e1d4-48d8-bed1-7535b77b3531)

Reply:
Done in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4262/commits/f67c4f3d5134ca8bc21be937b0d2dc4c30dddd95.

Reply:
Cool, thanks!
LGTM now!
<img width="932" alt="image" src="https://github.com/spinalcordtoolbox/spinalcordtoolbox/assets/39456460/e2774973-a5f1-41a4-b986-430f557ea8ca">